docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -68,21 +99,18 @@ The Alert details flyout contains the following informational tabs:
 [discrete]
 *Overview*
 
-The *Overview* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to analyze what generated the alert, so you can triage its relevance and resolve it as appropriate.
-
-The *Overview* tab has three main sections: *Reason*, *Document Summary*, and *Enriched data*. The *Reason* and *Document Summary* sections are standard for every alert. The *Enriched data* section is conditional and only displays for alerts with intelligence data.
+The *Overview* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to understand what generated the alert so you can appropriately triage and resolve it.
 
-The *Reason* section describes what generated the alert and provides general alert details. Use this information to understand the alert's origin and determine if the alert is relevant to your investigation.
-
-TIP: To view more details about the rule that produced the alert, click *View Rule detail page*.
+The *Overview* tab has three main sections: the summary alert summary, *Highlighted Fields*, and *Enriched data*. The alert summary and *Highlighted Fields* sections are standard for every alert. The *Enriched data* section only displays for alerts with intelligence data.
 
 [role="screenshot"]
-image::images/reason-section.png[Shows the Reason section of an alert]
+image::images/alert-details-flyout.png[Shows the alert details flyout]
 
-The *Document Summary* section provides general alert details. Use this information to investigate what generated the alert.
+The alert summary section provides general alert details such as the alert's status, the severity of the alert, the alert's calculated risk score, and a link to the rule that produced the alert. These details are provided for all alerts regardless of the alert type, and they offer an overview of important alert details.
 
-[role="screenshot"]
-image::images/document-summary-section.png[Shows the Document Summary section of an alert]
+The alert's reason statement is under the alert summary. The reason statement describes what generated the alert and provides general alert details. You can use this information to understand the alert's origin and determine if the alert is relevant to your investigation.
```

Comment:
Is it a reason section, or a description? You could be right, and I know it used to say reason, but to me they generally read like descriptions.

Reply:
I kept the original terminology because the statement matches the text shown in the Reason column when you view the alert from the Alert table. That said, I'm always open to representing it in a clearer way if "reason statement" is too ambiguous. @michaelolo24 what are your thoughts on continuing to use "reason statement" to describe the statement under the alert's main details? Should we continue to frame the text that way to align it with the Reason column in the Alerts table or change it to something else, like "alert description"?

Reply:
We should continue with the reason statement because it is that field in the table `kibana.alert.reason`, but also in the long run, we want to enhance this statement so keeping this naming will align with that